src/redux/reducers/index.js

```diff
@@ -1,8 +1,13 @@
 import { combineReducers } from 'redux';
 import data from 'redux/reducers/episodesReducer';
+import getCurrentPage from 'redux/reducers/paginationReducer';
 
 const rootReducer = combineReducers({
-  data
+  data,
+  getCurrentPage,
```

Comment:
please name it `currentPage`, names of keys of the store should be like a database keys

Reply:
Now your store looks weird with this key
![screen shot 2018-10-12 at 16 02 58](https://user-images.githubusercontent.com/16289193/46873991-58511780-ce38-11e8-890c-d52e78070db7.png)

Reply:
Right, I've changed the name.